bed/bedPe/bedPe.go

```diff
@@ -0,0 +1,185 @@
+// Package bedPe provides functions for handling Bed Paired End format files, used for
+// handling pairs of genomic regions.
+
+package bedPe
+
+import (
+	"fmt"
+	"github.com/vertgenlab/gonomics/bed"
+	"github.com/vertgenlab/gonomics/exception"
+	"github.com/vertgenlab/gonomics/fileio"
+	"io"
+	"log"
+	"strconv"
+	"strings"
+	"sync"
+)
+
+type BedPe struct {
+	Chrom1            string
```

Comment:
What do you think about ChromOne or ChromA.  I often have trouble reading code that does math, but also has numerals in the names.

Reply:
ChromOne in next commit, I was following the bedpe documentation names but i agree they are confusing